R/bind-event.R

```diff
@@ -244,8 +244,9 @@ bindEvent.shiny.render.function <- function(x, ..., ignoreNULL = TRUE, ignoreIni
     )
   }
 
-  class(res) <- c("shiny.render.function.event", class(res))
-  res
+  renderFunc <- addAttributes(renderFunc, renderFunctionAttributes(valueFunc))
+  class(renderFunc) <- c("shiny.render.function.event", class(valueFunc))
```

Comment:
Why is this necessary?

Reply:
It's so that we can throw an error if someone tries to call `bindCache` on it.